src/account-info/LogoutButton.js

```diff
@@ -27,7 +27,11 @@ class LogoutButton extends PureComponent {
   shutdownPUSH = async () => {
     const { auth, actions, pushToken } = this.props;
     if (pushToken !== '') {
-      await unregisterPush(auth, pushToken);
+      try {
+        await unregisterPush(auth, pushToken);
+      } catch (e) {
+        console.log('failed to unregister Push token', e); // eslint-disable-line
```

Comment:
Ideally we should hook up the sentry to log these error events, I'm not sure how to do it know, hence a log statement! 

It shouldn't happen though, the only param is the pushToken and this function is called the token is not ''